api/views/user_views.py

```diff
@@ -529,3 +532,34 @@ def post(self, request):
                 {"message": f"Invalid token: {str(e)}"},
                 status=status.HTTP_400_BAD_REQUEST,
             )
+
+
+class ContactPageView(APIView):
+    permission_classes = [AllowAny]
+
+    def post(self, request):
+        fname = request.data.get("first_name")
+        lname = request.data.get("last_name")
+        email = request.data.get("email")
+        message = request.data.get("message")
```

Comment:
Let's add a request serializer here, and the decorator that'll allow us to test the endpoint via Swagger

Reply:
Done